lib/hooks/useTVTheme.js

```diff
@@ -0,0 +1,15 @@
+import { useState, useEffect } from 'react'
+
+import useDarkMode from 'use-dark-mode'
+
+export const useTVTheme = () => {
```

Comment:
Is this function still in use?
and I think this function is exactly the same as useBgColor......
you can import useBgColor and rename it to useTVTheme if you want
`
import { useBgColor:  useTVTheme } from '@/lib/hooks/useBgColor'
`